Tests/Runner/DataLoaderTest.php

```diff
@@ -204,7 +205,7 @@ public function testLoadInputDataS3()
             $this->getS3StagingComponent(),
             new OutputFilter()
         );
-        $dataLoader->loadInputData();
+        $dataLoader->loadInputData(new InputTableStateList([]));
```

Comment:
@odinuv ještě dotaz - není divný, že ten argument je povinnej? na spoustě míst se tam prostě předává prázdnej objekt. stejně tak ta metoda vrací pokaždý zase novou instanci téhle třídy, když se nic nestane, tak s prázdnejma datama. viz třeba `NullDataLoader`

Reply:
neni to divny, nesnasim optional argumenty, je to semeniste chyb! ✝️